src/test/java/com/github/dockerjava/cmd/CreateContainerCmdIT.java

```diff
@@ -906,7 +906,7 @@ public void createContainerFromPrivateRegistryWithValidAuth() throws Exception {
 
     @Test
     public void createContainerFromPrivateRegistryWithNoAuth() throws Exception {
-        AuthConfig authConfig = RegistryUtils.runPrivateRegistry(dockerRule.getClient());
```

Comment:
I need variables in tests to have them in debugger, please don't apply cleanups in tests. Tests staff is for easier testing

Reply:
I understand, I will amend this PR tomorrow to remove the changes in the test code.